src/components/card/TemplateCard.vue

```diff
@@ -4,18 +4,22 @@
     tabindex="0"
     class="cursor-pointer template-card"
     :data-cy="`template-card_${template.key}`"
+    flat
   >
-    <q-img
-      :src="`/template-library/templates/${template.key}/icon.svg`"
-      height="100%"
-    >
+    <q-card-section class="q-pa-none">
+      <q-img
+        class="rounded-borders"
+        :src="`/template-library/templates/${template.key}/icon.svg`"
+      />
+    </q-card-section>
+    <q-card-section class="q-px-none q-py-sm">
       <div
```

Comment:
`div` not need anymore. Move its attribute to parent `q-card-section`.

Reply:
done